run.go

```diff
@@ -291,8 +298,17 @@ func transformRunOptions(options ...*RunOptions) (*RunOptions, error) {
 	}
 	if option.Stderr == nil {
 		option.Stderr = os.Stderr
+	}
+	if option.Logger == nil {
+		logger = slog.New(slog.NewTextHandler(option.Stderr, nil))
 	} else {
-		logger.SetOutput(option.Stderr)
+		logger = option.Logger
+	}
+
+	if option.CaptureAllOutputWithLogger {
```

Comment:
Hi @GuyGoldenberg, `stderr` of playwrightDriver is already redirected to `options.Stderr` in `newPipTransport`, and `stdout` only contains one-way communication protocol data. Is there a special reason to log the protocol output?